filebeat/docs/multiline.asciidoc

```diff
@@ -27,6 +27,19 @@ that span multiple lines.
 
 The following example shows how to configure {beatname_uc} to handle a multiline message where the first line of the message begins with a bracket (`[`).
 
+[source,yaml]
```

Comment:
Should we mention that this is the `filestream` syntax explicitly?

Reply:
Does this only work with filestream? If it does we should mention that, otherwise users might try to use it with the `log` input before reading the text below. Let's not assume users will read the entire page before trying anything.